src/include/distributed/commands.h

```diff
@@ -101,11 +101,11 @@ extern void ErrorIfUnsupportedForeignConstraintExists(Relation relation,
 													  uint32 colocationId);
 extern bool ColumnAppearsInForeignKeyToReferenceTable(char *columnName, Oid
 													  relationId);
-extern List * GetTableForeignConstraintCommands(Oid relationId);
-extern bool HasForeignKeyToReferenceTable(Oid relationId);
-extern bool TableReferenced(Oid relationId);
-extern bool TableReferencing(Oid relationId);
-extern bool ConstraintIsAForeignKey(char *constraintName, Oid relationId);
+extern List * GetForeignConstraintCommandsTableReferencing(Oid relationOid);
```

Comment:
The  typical convention is to use `relationId`. Do you think  `relationOid` is better?

Reply:
I guess we switch between using either of them in code base. Actually, `relationOid` seems a bit convenient to me as type here is  an `Oid` itself. What do you think ?

Reply:
```
RaymondT495:citus onderkalaci$ grep -Ri "relationId" src/backend/distributed/ | wc -l
    1856
RaymondT495:citus onderkalaci$ grep -Ri "relationOid" src/backend/distributed/ | wc -l
     202
```

Data suggests that relationId is a lot more common. And, note that this is on your branch, on  the master, the difference  is bigger.